.github/workflows/imports.yml

```diff
@@ -0,0 +1,49 @@
+# Run python scripts to import/sync data from external sources
+
+name: Data imports
+
+on:
+  workflow_dispatch:
+    inputs:
+      Task:
+        type: choice
+        description: "The task to perform"
+        required: true
+        options:
+          - 'hubspot'
+      Environment:
+        type: choice
+        description: "The environment to target"
+        required: true
+        default: 'qa'
+        options:
+          - 'qa'
+          - 'prod'
+      Destructive:
+        description: Check to confirm you are happy to proceed with a destructive operation
+        type: boolean
+        required: false
+        default: false
+
+jobs:
+  check_for_danger:
```

Comment:
I reckon having the chekbox it good regardless? It's doing something in production.

We can always remove it.